src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -104,27 +104,28 @@
     "rxjs": "6.6.3",
     "simplebar-angular": "2.3.0",
     "swagger-ui": "3.49.0",
-    "tslib": "2.0.2",
+    "tslib": "^2.0.0",
     "zone.js": "0.10.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "0.1002.3",
-    "@angular/cli": "10.1.6",
-    "@angular/compiler-cli": "10.1.5",
-    "@angular/language-service": "10.1.5",
+    "@angular-devkit/build-angular": "0.1102.14",
+    "@angular/cli": "11.2.14",
+    "@angular/compiler-cli": "11.2.14",
+    "@angular/language-service": "11.2.14",
     "@applitools/eyes-cypress": "^3.22.0",
     "@compodoc/compodoc": "1.1.11",
     "@types/jest": "26.0.14",
     "@types/lodash": "4.14.161",
     "@types/node": "12.12.62",
     "@types/simplebar": "5.1.1",
-    "@types/swagger-ui": "3.47.0",
-    "codelyzer": "6.0.1",
+    "@types/swagger-ui": "3.52.0",
+    "codelyzer": "^6.0.0",
```

Comment:
@nizamial09 I know it's too late but we use pinned versions for stability purposes.

Reply:
@alfonsomthd Oh okay, I think its the angular migration script's doing.  But I can change it here maybe https://github.com/ceph/ceph/pull/43404 (just something I started on weekend ;) )